crypto/perlasm/x86_64-xlate.pl

```diff
@@ -109,14 +112,16 @@
 
 { package opcode;	# pick up opcodes
     sub re {
-	my	$self = shift;	# single instance in enough...
-	local	*line = shift;
-	undef	$ret;
+	my	$class = shift;
+	my	$self = {};
+	my	$line = shift;
```

Comment:
"This applies to multiple cases..." means that *all* occurrences should be harmonized. Go for `my (...) = @_`. As for `register::re`, it's not like script sets styling standard. Though massive changes are not super-desirable, because one wants to keep changes so to say "back-portable". There is no sharp line though...

P.S. I'm signing off for today, so further replies will come with delay...